harness/determined/cli/experiment.py

```diff
@@ -578,6 +578,35 @@ def experiment_logs(args: Namespace) -> None:
         )
 
 
+def set_log_retention(args: Namespace) -> None:
+    if not args.forever and not isinstance(args.days, int):
+        raise cli.errors.CliError(
+            "Please provide an argument to set log retention. --days sets the number of days to"
+            + " retain logs from the time of creation, eg. `det e set log-retention 1 "
+            + "--days 50`. --forever retains logs indefinitely, eg.`det e set log-retention 1"
+            + "--forever."
```

Comment:
missing a space, this renders as `det e set log-retention 1--forever`

Reply:
fixed